web-client/terraform/bin/deploy-app.sh

```diff
@@ -28,25 +35,24 @@ else
   echo "dynamodb lock table already exists"
 fi
 
-# exit on any failure
 set -eo pipefail
 
-DYNAMSOFT_URL="https://dynamsoft-lib.${EFCMS_DOMAIN}"
+DYNAMSOFT_URL="https://dynamsoft-lib-${ENVIRONMENT}.${EFCMS_DOMAIN}"
```

Comment:
hm, are we sure about this one? I think everyone so far is keying off of our staging environment. this would appear to necessitate a change in the `DYNAMSOFT_URL_OVERRIDE`. I'm going to go back and check the terraform for these dynamsoft urls.

Reply:
I believe you are correct. this should be reverted back.

Reply:
Resolved in...
- `web-client/terraform/bin/deploy-app.sh`
- `web-client/terraform/bin/deploy-init.sh`
- `web-client/terraform/bin/destroy-app.sh`